Source/Core/Common/JsonUtil.cpp

```diff
@@ -5,8 +5,33 @@
 
 #include <fstream>
 
+#include <fmt/format.h>
+
 #include "Common/FileUtil.h"
 
+namespace
+{
+template <typename M>
+picojson::object MatrixToJson(const M& mat)
+{
+  picojson::object obj;
+  for (std::size_t i = 0; i < mat.data.size(); i++)
+  {
+    obj.emplace(fmt::format("m{}", i), mat.data[i]);
```

Comment:
Is this doing `{'m0': 0.0, 'm1': 1.0, 'm2': 1.0, ...}` ?
Would it make more sense as an array?